pages/index.tsx

```diff
@@ -7,10 +8,20 @@ import EventBanner from '../components/EventBanner';
 import HeroBanner from '../components/HeroBanner';
 import Footer from '../components/Footer';
 import Section from '../layouts/Section';
+import HackathonModal from '../components/HackathonModal';
+import Modal from 'react-modal';
+
+Modal.setAppElement('#__next'); // Set the app element for accessibility
```

Comment:
This could be est in `HackathonModal.tsx` right? I am thinking we should keep it as self contained as possible. (I am hoenstly not the biggest fan of `react-modal` having a global state like this tbh.